infrastructure/azure.tf

```diff
@@ -850,3 +851,28 @@ module "kubernetes" {
   service_principal_client_id = "${data.azurerm_client_config.current.client_id}"
   service_principal_client_secret = "${var.ARM_CLIENT_SECRET}"
 }
+
+#
+# Allocates a public IP for exposing the Kubernetes services
+#
+# This IP needs to be registered on the following CNAMEs:
+#
+# *.k8s.test.cd.teamdigitale.it (for the test environment)
```

Comment:
is this going to change with *italia.it someday ?

Reply:
You mean the IP? it's up to us to decide, until we delete the Azure resource we can keep using that IP.

If we have a single Kubernetes cluster for all environments we could even use a single IP to route traffic to pods in different namespaces/environments.